frontend/packages/ux-editor-v3/src/components/config/FormComponentConfig.test.tsx

```diff
@@ -6,7 +6,7 @@ import { componentMocks } from '../../testing/componentMocks';
 import InputSchema from '../../testing/schemas/json/component/Input.schema.v1.json';
```

Comment:
```suggestion
import InputSchema from '@studio/testing/schemas/json/component/Input.schema.v1.json';
```

Reply:
Same here as in [my previous comment](https://github.com/Altinn/altinn-studio/pull/12797#discussion_r1611766890)